storage/src/mmr/verification.rs

```diff
@@ -337,19 +337,19 @@ impl<H: CHasher> Proof<H> {
     }
 }
 
-async fn peak_digest_from_range<'a, H, M: Hasher<H>, I, S>(
-    hasher: &mut M,
+async fn peak_digest_from_range<'a, InnerHasher, H: Hasher<InnerHasher>, I, S>(
```

Comment:
You used just "I" for inner hasher earlier, would rather you make this consistent here and use something else for the iterator.